.github/workflows/prepare-release.yml

```diff
@@ -143,10 +143,27 @@ jobs:
           echo "release-version=${CURRENT_VERSION}" >> $GITHUB_OUTPUT
           echo "next-version=${NEXT_VERSION}" >> $GITHUB_OUTPUT
 
+      - name: Read extension metadata
+        id: extension-metadata
+        run: |
+          EXTENSION_METADATA_PATH=runtime/target/classes/META-INF/quarkus-extension.yaml
```

Comment:
I don't think that's going to fly, there are some repositories with more than one extension.

What if we read this information from the `.github/project.yaml`? Perhaps an `additional-notes` entry there would work?